tests/PriorityMapTest.php

```diff
@@ -89,23 +91,42 @@ public function testGetKeys()
         $list->set('k4', 'v4', 1);
         $list->set('k5', 'v5', 4);
 
-        $this->assertEquals(
+        $result = $list->getKeys();
+
+        $this->assertSame(
             array('k1', 'k2', 'k3', 'k4', 'k5'),
             $list->getKeys()
         );
     }
 
+    public function testCount()
+    {
+        $list = new PriorityMap();
+
+        $list->set('K1', 'v1', 2);
+
+        $this->assertSame(1, $list->count());
+    }
+
+    public function testCountWithEmptyList()
+    {
+        $list = new PriorityMap();
+
+        $this->assertSame(0, $list->count());
+    }
+
     public function testGetKeys_EmptyList()
     {
         $list = new PriorityMap();
 
-        $this->assertEquals(
-            array(),
-            $list->getKeys()
-        );
+        $this->assertSame([], $list->getKeys());
```

Comment:
short array syntax awailable only since PHP 5.4, you broke backward compatibility